lms/djangoapps/email_marketing/models.py

```diff
@@ -28,7 +28,7 @@ class Meta(object):
     )
 
     sailthru_new_user_list = models.fields.CharField(
-        max_length=48,
+        max_length=512,
```

Comment:
@ahsan-ul-haq I think updating the existing column for new user list on Sailthru is dangerous. Since you need to data migrate, it is possible during the data migration, we may have user registered not able to be determined on which email list to be saved to. Do you think it would be better to add a new column with the json value, so we can update the code using the new column? After the change is deployed, we can create a new PR to retire the existing "sailthru_new_user_list" column. What do you think?